Gemfile

```diff
@@ -88,6 +88,7 @@ gem "sentry-rails"
 gem "discard"
 
 # Geocoding
+gem "audited"
 gem "geocoder"
```

Comment:
I don't think this should sit underneath the comment `# Geocoding` since it's unrelated.

```suggestion
# Geocoding
gem "geocoder"

# Audit trail
gem "audited"
```